Bottom: 271e4683573914f46c32dd88b4c04e2c7cb3511d
Top:    f61a5824d2425e408d3b4705df4e08dc919e90bf
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-09 08:54:25 +0000

Avoid clearing pins / vias from polygons... it is already done!







---

diff --git a/src/draw.c b/src/draw.c
index 545c6e9..dd4ff4d 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1262,6 +1262,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
       AddPart (Pin);
       return;
     }
+#if 0
   /* Clear the area around the pin */
   if (TEST_FLAG (SQUAREFLAG, Pin))
     {
@@ -1284,6 +1285,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
     {
       gui->fill_circle (Output.pmGC, Pin->X, Pin->Y, half);
     }
+#endif
   if ((!TEST_FLAG (PINFLAG, Pin) && !PCB->ViaOn)
       || (TEST_FLAG (PINFLAG, Pin) && !PCB->PinOn))
     return;
